src/main/java/com/oltpbenchmark/benchmarks/tpch/generation/Distribution.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oltpbenchmark.benchmarks.tpch.generation;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.base.Preconditions.checkState;
```

Comment:
We should probably remove this.

Reply:
I am not sure I have resolved this. Do you mean only the checkState and toStringHelper should be removed, or all com.google.common should be avoided?

Reply:
Do we already include `com.google.common` packages in maven? I don't want additional dependencies just for these checks.

Reply:
Not changing the pom.xml. The repo compiles.

The pom has a com.google.cloud pakage which fetches all google.common so no additional dependency needed.
Yet strangely, google packages are used nowhere except for the TPCH code I have introduced.

Reply:
So the google package functionalities may be replaced by self-defined ones, and com.google.cloud can be instead deleted (google dependency tree is big).

Reply:
resolved this. no google pkg needed.